packages/appium/lib/logsink.js

```diff
@@ -186,8 +188,29 @@ async function createTransports(args) {
   return transports;
 }
 
+let prefixesEncountered = [];
+
+function getPrefixColor(prefixId) {
+  try {
+    if (prefixesEncountered.indexOf(prefixId) > -1) {
+      return '\x1b[38;5;' + (prefixesEncountered.indexOf(prefixId) * 16) + 'm';
+    }
+    prefixesEncountered.push(prefixId);
+    const colorNumber = (nextColorIndex * 16);
+    nextColorIndex++;
+    if (colorNumber > 256) {
+      return '';
+    }
+    const ansiColorCode = '\x1b[38;5;' + (colorNumber) + 'm';
+    return ansiColorCode;
+  } catch (err) {
+    return '';
+  }
+}
```

Comment:
@jlipps @boneskull I can probably clean up this code a little if the strategy of using the ANSI color code is acceptable.